sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,118 @@
+| `base_frame`                                  | `std::string`         | `base_link`                      | The base_frame is used for visualization.                                                                                                                 |
+| `min_tag_size`                                | `double`              | `0.6`                            | The size of the AprilTag in meters (payload).                                                                                                             |
+| `max_tag_distance`                            | `double`              | `20.0`                           | Maximum allowed distance in meters from the camera to the tags.                                                                                           |
+| `max_allowed_homography_error`                | `double`              | `0.5`                            | AprilTag detection are discarded if the homography error is larger than `max_allowed_homography_error`.                                                   |
+| `use_receive_time`                            | `bool`                | `false`                          | Flag to determine whether to use the receive time instead of the header timestamps.                                                                       |
+| `use_rectified_image`                         | `bool`                | `true`                           | Flag to determine whether rectified images should be used in the calibration process.                                                                     |
+| `calibration_crossvalidation_training_ratio`  | `double`              | `0.7`                            | The ratio of data used for training versus testing during the calibration's cross-validation process.                                                     |
+| `calibration_convergence_min_pairs`           | `int`                 | `9`                              | The minimum number of AprilTag and LidarTag detection pairs required to consider the calibration process as potentially converged.                        |
+| `calibration_convergence_min_area_percentage` | `double`              | `0.005`                          | Minimum percentage of the area that needs to be covered by detection.                                                                                     |
+| `min_pnp_points`                              | `int`                 | `8`                              | Minimum number of points required for the Perspective-n-Point problem used in calibration to solve the pose estimation.                                   |
+| `min_convergence_time`                        | `double`              | `6.0`                            | Minimum time required for the calibration process to be considered as converged.                                                                          |
```

Comment:
The description for `min_convergence_time` is wrong

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/35362526d946236ad4f54d91cf0fda99f20bcf57